core/shared/src/main/scala/coursier/util/Parse.scala

```diff
@@ -5,6 +5,7 @@ import coursier.core.{Module, Repository}
 import coursier.ivy.IvyRepository
 import coursier.maven.MavenRepository
 
+import java.net.URLDecoder.decode
```

Comment:
Couldn't we do the `decode` part only from where the `"url"` value is used, that is in the `cli` module for now? That way, we wouldn't have to do an `encode` for it in the tests. (`URLEncoder.encode` seems not to be available from scala-js, hence the current CI errors, so that would solve that to.)